fetch-validator-status/networks.json

```
@@ -1,66 +1,82 @@
 {
   "sbn": {
     "name": "Sovrin Builder Net",
+    "indyNamespace": "sovrin:builder",
     "genesisUrl": "https://raw.githubusercontent.com/sovrin-foundation/sovrin/stable/sovrin/pool_transactions_builder_genesis"
   },
   "ssn": {
     "name": "Sovrin Staging Net",
+    "indyNamespace": "sovring:staging",
```

```suggestion
    "indyNamespace": "sovrin:test",
```

Sovrin StatgingNet was recently renamed to Sovrin TestNet.

---

Fixed